algorithms/polish/stemmer.tt

```diff
@@ -0,0 +1,166 @@
+[% header('Polish stemming algorithm') %]
+
+<h2>Links to resources</h2>
+
+<ul>
+[% algorithm_lis('polish', 'Polish') %]
+</ul>
+
+[% algorithm_vocab([20, 'babicka', 'maaru']) %]
+
+<h2>The stemming algorithm</h2>
```

Comment:
What's the origin of the algorithm?

If there's a paper or a description of it somewhere, it's helpful to reference it.

If it's just something you devised, that's OK but it's useful to say that (and really you should get credit for creating it).